jupyterhub/files/hub/jupyterhub_config.py

```diff
@@ -276,6 +276,7 @@ def camelCaseify(s):
         {
             "name": volume_name_template,
             "persistentVolumeClaim": {"claimName": pvc_name_template},
+            "subPath": get_config("singleuser.storage.dynamic.subPath"),
```

Comment:
subPath is something you declare in a volumeMount and not part of the volume itself

I considered suggesting we could name this volumeMountSubPath or similar to help users map this better to the k8s implementation, but its also named specifically `subPath`.

Reply:
My mistake. Moved it to volumeMount.